package.json

```diff
@@ -25,6 +25,7 @@
     "nextra-theme-docs": "latest",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-gtm-module": "^2.0.11",
```

Comment:
Fine for now but do note this library isn't great with later versions of nextjs; in particular once I get us onto nextjs 14 and the app router across the board we will need to use something that leverages the nextjs `Script` component instead since it properly handles blocking / prioritization and CSP.

I don't know of any libraries that implement this out of the box but it's really trivial to do it manually -- I'll add it in after I migrate us.